onnxruntime/core/optimizer/graph_transformer_factory.h

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/optimizer/graph_transformer.h"
+#include "core/optimizer/conv_activation_fusion.h"
+#include "core/optimizer/conv_add_fusion.h"
+#include "core/optimizer/identity_elimination.h"
+#include "core/optimizer/slice_elimination.h"
+#include "core/optimizer/unsqueeze_elimination.h"
+#include "core/optimizer/conv_bn_fusion.h"
+#include "core/optimizer/conv_mul_fusion.h"
+#include "core/optimizer/gemm_activation_fusion.h"
+#include "core/optimizer/matmul_add_fusion.h"
+#include "core/optimizer/insert_cast_transformer.h"
+
+using namespace ::onnxruntime::common;
+
+namespace onnxruntime {
+/*
+* Owns all the graph transformers.
+*/
```

Comment:
Usually a factory creates things and passes on ownership. Is this more of a GraphTransformerRegistry?

Reply:
right I will rename this as registry

Reply:
Is it necessary to define a new class to own the transformers? Why not just use GraphTransformerManager? GraphTransformerManager already provided a register method.